package.json

```diff
@@ -40,6 +40,11 @@
     "url": "https://github.com/openmrs/openmrs-esm-login/issues"
   },
   "dependencies": {
+    "@carbon/colors": "^10.19.0",
+    "@ibm/plex": "^5.1.3",
+    "carbon-components": "^10.27.0",
+    "carbon-components-react": "^7.27.0",
+    "carbon-icons": "^7.0.7",
```

Comment:
This is one of the shared dependencies. No need to include it.

Reply:
@FlorianRappl  you mean  "carbon-icons": "^7.0.7" ?

Reply:
Yes, always the line that I am commenting on.

Reply:
I was confused by the fact that it also included   `"carbon-components-react": "^7.27.0",`